Documentation/Makefile

```diff
@@ -27,6 +27,7 @@ check-requirements:
 	  exit 1)
 
 docs-container:
+	cd .. && \
```

Comment:
Can we move this target to the top level Makefile instead?

Reply:
fixed